carts/views.py

```diff
@@ -32,3 +32,32 @@ def get(self, request):
 
         except Cart.DoesNotExist:
             return JsonResponse({"message" : "CART_NOT_EXIST"}, status = 404)
+    
+    @login_decorator
+    def patch(self, request):
+        try:
+            cart_id  = request.GET.get('cart_id')
+            data     = json.loads(request.body)
+            quantity = data['quantity']
+
+            if cart_id:
+                Cart.objects.filter(id = int(cart_id)).update(quantity = quantity)
+        
+            return JsonResponse({"message" : "SUCCESS"}, status = 201)
+        
+        except Cart.DoesNotExist:
+            return JsonResponse({"message" : "INVALID_CART"}, status = 404)
+
+    @login_decorator
+    def delete(self, request):
+        try:
+            cart_id  = request.GET.get('cart_id')
+            
+            if cart_id:
+                Cart.objects.filter(id = int(cart_id)).delete()
```

Comment:
```suggestion
            cart = Cart.objects.get(user=request.user, cart_id=cart_id)
            cart.delete()
```
cart를 먼저 조회하는데, cart id로만 조회 하는 것이 아니고, user까지 포함 해줘야 그 유저가 가지고 있는 cart를 정확하게 특정 할 수 있습니다. 그리고 filter보다는 정확히 특정 지을 수 있는 get이 더 좋을 것 같습니다.(아래 예외처리도 하셨고요.)

Reply:
넵 get으로 변경해서 해당 user까지 담아주었습니다.